### ISSUE TITLE
redirect problem

### ISSUE THREAD

**Issue by mrac** [zalando-incubator/solution-center-login#2]
**Problem:**

I'm not sure if I use it correctly, but I'm testing `redirectToHomeIfAuthenticated()` and `redirectToLogin()` function for 'STAGING' but locally, with `setSolutionCenterCommunication(false)`, and I got redirected to:
`
http://localhost:7777/usf-stage.norris.zalan.do/#/
`
or to:
`
http://localhost:7777/usf-stage.norris.zalan.do/#/login?redirect=https://usf-stage.norris.zalan.do
`

**Probable cause:**

It looks like this line:
`$window.location.href = environmentsService.getSolutionCenterUrl(self.getEnvironment()) + "/#" + redirectionPath;
`
gives relative URL instead of absolute one?

**Comment by dami-gg** [comment]
The idea behind the environment configuration is basically to avoid mixing environments so because of that you can't use STAGING mode from localhost.

By the way, you don't need to set the flag related to the Solution Center communication since its default value is false.

**Comment by mrac** [comment]
All right, thanks! I'll deploy to stage and test it there :)

**Comment by cjhowald** [comment]
Also, you shouldn't need to call the redirect functions directly. Just call into `requireAuthenticatedUser` and it will kick off the authentication flow with the proper logic.

**Comment by cjhowald** [comment]
We protect our views requiring authentication with a resolve in the router like so:

```javascript
resolve: {
  tempuser : function(authenticationService) {
    return authenticationService.requireAuthenticatedUser();
  }
}
```

Note that the documentation is incomplete but will be updated to include this.

**Comment by mrac** [comment]
Actually it happens also for 'LOCAL'.

```
$window.location.href = environmentsService.getSolutionCenterUrl(self.getEnvironment()) + "/#" + redirectionPath; 
```
seems to set the relative URL, not absolute (without http://).

**Comment by mrac** [comment]
It's solved in ver 0.1.4. Thanks!
